psalm.baseline.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files>
+  <file src="tests\Unit\Utilities\CssConcatenatorTest.php">
+    <PossiblyNullPropertyAssignmentValue occurrences="1">
```

Comment:
iirc, this particular issue relates to a class property being set in `TestCase::setUp()` rather than a constructor.

what I can't recall is whether or not that particular scenario was resolved in a later release of psalm/plugin-phpunit.

tl:dr; update the baseline when you're able to upgrade psalm/plugin-phpunit to see if the issue goes away.

Reply:
> Maybe add an XML comment to that effect?

I'd thought of that, but I'm pretty sure the XML comment would be destroyed whenever psalm wrote to the baseline.

Reply:
Oh, so does Psalm potentially also write to this file (which looks like a config file to me)?  If so, we could nonetheless re-insert the comment when it does (or simply reject that specific change when doing a diff).

Reply:
It does (if the corresponding command line argument is provided). I highly recommend not editing this file manually as it is supposed to be automatically created.

Reply:
> I highly recommend not editing this file manually as it is supposed to be automatically created.

If it's automatically created, then we should not be adding this file to the repository at all.  Or am I missing something?

Reply:
> > I highly recommend not editing this file manually as it is supposed to be automatically created.
> 
> If it's automatically created, then we should not be adding this file to the repository at all. Or am I missing something?

running `psalm --update-baseline` or `psalm --set-baseline=./psalm.baseline.xml` will cause the file to be written to.

the bugfix commits on this PR were created by working from the bottom up, removing an issue type or code entry, then running psalm to find the code that caused that chunk of the baseline to be added.

Reply:
> If so, we could nonetheless re-insert the comment when it does (or simply reject that specific change when doing a diff).

this method is also consistent with the manual changes performed first in 130782e (has to be re-done every time with psalm 3.2 to avoid travis failing) and in 2b6ec42 (doesn't affect build status, but is still annoying).

Reply:
resolved in af615a9, flagged to be monitored by #793